### PR TITLE
API/server: add README file.

### PR DESCRIPTION
**Pull request by mgolosova**

### CODE REVIEW

**Comment by mgolosova** [comment]
> @mgolosova [force-pushed](https://github.com/PanDAWMS/dkb/compare/e0575de795d9cce1535daeacf9b227767530aabc..7e7583f5b09e57fa4d260dcf63397fe7807a4341) the api-readme branch from e0575de to 7e7583f

Minor codestyle fix (trailing space).

File: Utils/API/server/README [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                  - file name may be altered via parameter -s|--sock;
+ - %%ADDR%%    -- bind address (-l|--listen);
+ - %%WWW_DIR%% -- installation directory for the application (-d|--dest);
+ - %%CFG_DIR%% -- directory with app's configuration files (-c|--cfg-dir).
+
+These special strings may be used in the templates of system files and in the
+application files (those listed in .files).
+
+
+2. Installation copies built files to system directories. Can manage (or not)
```
Comment (witxka) [inline Utils/API/server/README:100]: "Can manage (or not) any combinations of"  I would rephrase with something like "Any combination of the following command
line options can be used:"

Comment (mgolosova) [reply Utils/API/server/README:100]: @witxka,
My wording is poor, you're right... but the idea was to list not the options, but what can be managed by this setup script.  I mean, the "WWW application files" are not the "command line option(s)." Will try to reword...

File: Utils/API/server/README [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+ - WWW application files   -- (-w|-www), (-W|--no-www): managed by default, but
+                                                        can be disabled;
+ - Nginx configuration     -- (-n|--nginx);
+ - Systemd configuration   -- (-S|--service);
+ - SELinux customisaton    -- (--sel): make sure that WWW application execution
+                                       is allowed.
+
+
+There are two main steps of the installation process: building and installaton
```
Comment (Evildoor) [inline Utils/API/server/README:94]:
```suggestion
There are two main steps of the installation process: building and installation
```

**Comment by mgolosova** [comment]
@Evildoor, thank you, missed this one.
